README.md

```diff
@@ -0,0 +1,284 @@
+
+# History
+
+The package was originally conceived by Jonathan Shimwell and based on the
+[FreeCAD Python API](https://wiki.freecadweb.org/FreeCAD_API). When 
+[CadQuery 2](https://github.com/CadQuery/cadquery) was released the project
+started to migrate the code base. Shortly after this migration the project
+became open-source and has flourished ever since. The project has grown largely
+due to two contributors in particular (John Billingsley and
+Remi Delaporte-Mathurin) and others have also helped, you can see all those who
+have helped the development in the 
+[Authors.md](https://github.com/ukaea/paramak/blob/main/AUTHORS.md) and in the 
+[github contributions](https://github.com/ukaea/paramak/graphs/contributors). 
+
+
+## System Installation
+
+To install the Paramak you need to have 
+[Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/), 
+[Cadquery 2](https://cadquery.readthedocs.io/en/latest/installation.html) and 
+[Pip](https://anaconda.org/anaconda/pip). If you have these three dependencies 
+already then you can install the Paramak using Pip:
+
+```
+pip install paramak
+```
+
+Detailed installation 
+[instructions](https://paramak.readthedocs.io/en/main/#prerequisites)
+can be found in the User's Guide.
+
+
+## Docker Image Installation
+
+Another option is to use the Docker image which contains all the required
+dependencies.
+
+1. Install Docker CE for
+[Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/),
+[Mac OS](https://store.docker.com/editions/community/docker-ce-desktop-mac), or
+[Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows),
+including the part where you enable docker use as a non-root user.
+
+2. Pull the docker image from the store by typing the following command in a
+terminal window, or Windows users might prefer PowerShell.
+
+    ```docker pull ukaea/paramak```
+
+3. Now that you have the docker image you can enable graphics linking between
+your os and docker, and then run the docker container by typing the following
+commands in a terminal window.
+
+    ```sudo docker run -p 8888:8888 ukaea/paramak /bin/bash -c "jupyter notebook --notebook-dir=/opt/notebooks --ip='*' --port=8888 --no-browser" --allow-root```
+
+4. A URL should be displayed in the terminal and can now be opened in the
+internet browser of your choice. This will load up the examples folder where
+you can view the 3D objects created. Alternatively the Docker imag can be run
+in terminal mode ```docker run -it ukaea/paramak```
+
+Alternatively the Docker image can be run in terminal mode .
+```docker run -it ukaea/paramak```
+
+You may also want to make use of the
+[--volume](https://docs.docker.com/storage/volumes/)
+flag when running Docker so that you can retrieve files from the Docker
+enviroment to your base system.
+
+Docker can also be used to run the tests using the command
+```docker run -rm ukaea/parama pytest tests```
```

Comment:
**MAINTAINABILITY**

1. There are several typos in this line:
* `-rm` -> `--rm`
* `ukaea/parama` -> `ukea/paramak`

2. When the command is run, it cannot find the `tests` directory:

```
$ docker run --rm ukaea/paramak pytest tests
============================= test session starts ==============================
platform linux -- Python 3.7.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
rootdir: /
plugins: cov-2.10.1
collected 0 items

============================ no tests ran in 0.00s =============================
ERROR: file or directory not found: tests

```

This seems to be because the docker container spits us out in `/examples` rather than `/` because I can run the tests manually from inside the docker container:

```
[15:19:11][michael]:~/review/paramak$ docker run --rm -ti ukaea/paramak 
(base) root@07c6f4d85ce9:/examples# pwd
/examples
(base) root@07c6f4d85ce9:/examples# cd /
(base) root@07c6f4d85ce9:/# pytest tests
=================================== test session starts ===================================
platform linux -- Python 3.7.6, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
rootdir: /
plugins: cov-2.10.1
collected 362 items                                                                       

tests/test_Reactor.py .................................                             [  9%]
tests/test_Shape.py ...............................                                 [ 17%]
tests/test_example_components.py .....                                              [ 19%]
tests/test_example_reactors.py ...                                                  [ 19%]
...
====================== 362 passed, 228 warnings in 402.34s (0:06:42) ======================
```
<p align="right">
    <a href="https://www.pullrequest.com/">
        <img width="15%" src="https://static.pullrequest.com/images/dashboard/pullrequest-logo.svg">
    </a>
</p>

Reply:
fixed the typos